tools/packaging/boms/mac.json

```diff
@@ -347,6 +347,22 @@
     "Pattern": "Modules/PSReadLine/*.txt",
     "FileType": "NonProduct"
   },
+  {
+    "Pattern": "Modules/Microsoft.PowerShell.ThreadJob/*.dll",
+    "FileType": "NonProduct"
+  },
+  {
+    "Pattern": "Modules/Microsoft.PowerShell.ThreadJob/*.psd1",
+    "FileType": "NonProduct"
+  },
+  {
+    "Pattern": "Modules/Microsoft.PowerShell.ThreadJob/Microsoft.PowerShell.ThreadJob.pdb",
```

Comment:
We should avoid shipping the pdb.  Can we file an issue in the threadjob repo

Reply:
Issue filed in the [PowerShell/ThreadJob repo](https://github.com/PowerShell/ThreadJob/issues/28)
Looking into the repo and it's structure to see if can get a better understanding and potentially raise a PR myself for this